src/main/java/it/pagopa/pn/deliverypush/service/impl/NotificationCancellationServiceImpl.java

```diff
@@ -61,8 +61,8 @@ public Mono<StatusDetailInt> startCancellationProcess(String iun, String paId, C
                 .doOnError(err -> logEvent.generateFailure("Error in cancellation process iun={} paId={}", iun, paId, err).log());
     }
 
-    public void completeCancellationProcess(String iun){
-        log.debug("Start cancelNotification - iun={}", iun);
+    public void continueCancellationProcess(String iun){
+        log.debug("Start continueCancellationProcess - iun={}", iun);
         PnAuditLogEvent logEvent = generateAuditLog(iun, SECOND_CANCELLATION_STEP);
```

Comment:
AuditLog da modificare (lo segno solo per verificare che sia stato fatto in qualche altro branch)

Reply:
Risolvo in quanto ho verificato la modifica su altro branch